e2e/common/common.steps.ts

```diff
@@ -86,6 +102,27 @@ class CommonSteps {
       .to.eventually.be.true.notify(callback);
   }
 
+  @then(/^she is presented with the "([^"]*)" elements*$/)
+  public expectElementsPresent(elementClassNames: string, callback: CallbackStepDefinition): void {
+
+    const elementClassNamesArray = elementClassNames.split(',');
```

Comment:
btw there is lookup ready for us by protractor: `by.cssContainingText()`
Exmaple: https://github.com/redhat-ipaas/ipaas-ui/blob/afae5ceb2d3849f55528e03f24387d6704eb519c/e2e/integrations/edit/edit.po.ts#L52